entrypoint.sh

```diff
@@ -80,12 +87,11 @@ then
       pre_tag="$initial_version"
     fi
 else
-    log=$(git log $tag..HEAD --pretty='%B')
+    log=$(git log $tag..HEAD --pretty='%B' || git log "v$tag..HEAD" --pretty='%B')
+    # get current commit hash for tag
+    tag_commit=$(git rev-list -n 1 $tag)
```

Comment:
This might also failed if the real tag is `v` prefix because `$tag` is always without prefix.

Reply:
you are correct: 
because the script does not use `set -e` I'm surprized this works at all :)
here is a run and a line where this exact thing happened: https://github.com/alexeyatbluescape/test-actions/runs/6292861267?check_suite_focus=true#step:5:87

script did not fail, but it will not be able to detect skip logic correctly...